### PR TITLE
use explicit coin, base. consolidate ticker info to interface layer

### LINKED ISSUE #47: Ticker Standardization for Exchange Agnostic Instatiation
As a (GUI) user, I'd like to supply Alt and Base coins separately (explicitly).

Tux exchange tickers look backwards, and the different exchanges use different schemes.

Can we have Merkato() accept this: Merkato(..., ..., alt="LTC", base="BTC")

the interface level would translate to the specific exchange ticker ID > "BTC_LTC"

### PR DESCRIPTION
**Pull request by nasaWelder**
also added get_[lowest_bid | highest_ask | last_trade_price] to exchange layer

would close #47 Ticker Standardization issue

### CODE REVIEW

**Comment by nasaWelder** [comment]
note i did not put coin and base into config because config should be shared between pairs on the same exchange to reduce copies of keys in db

**Comment by mark-antony1** [comment]
We currently do not use the merkato/exchanges/exchange.py file, the current structure has the Merkato object accessing the tux exchange directly for example.

**Comment by nasaWelder** [comment]
Fffffffff ok I'll fix it tomorrow.

**Comment by nasaWelder** [comment]
Yikes:
from merkato.exchanges.tux_exchange.exchange import TuxExchange

**Comment by nasaWelder** [comment]
Two things:
1. I think this PR still works, since exchange.py was just pass through.

2. Why are we hardcoding TuxExchange into merkato.exchange object. The old way allowed bolting on new exchanges.

File: merkato/exchanges/tux_exchange/exchange.py [hunk 1]
```diff
@@ -13,11 +13,59 @@
...
+    def __init__(self, config, coin, base):
         self.privatekey = config['privatekey']
         self.publickey  = config['publickey']
         self.limit_only = config['limit_only']
         self.retries = 5
+        self.coin = coin
+        self.base = base
+        self.ticker = self.translate_ticker()
+
+    def translate_ticker(self):
```
Comment (mark-antony1) [inline merkato/exchanges/tux_exchange/exchange.py]: can you extract this to a utility function?

**Comment by mark-antony1** [comment]
It is hard coded from now, but there will be a function that merkato.py calls upon initialization that takes the configuration object and decides which exchange class to use. 

The merkato.exchanges.exchange.py is deprecated and will not be used as far as I know, but everything else looks great!

File: merkato/exchanges/tux_exchange/exchange.py (hunk 1 above)
Comment (nasaWelder) [reply merkato/exchanges/tux_exchange/exchange.py]: Not sure I understand, where to? TuxExchange/utils.py?

Comment (mark-antony1) [reply merkato/exchanges/tux_exchange/exchange.py]: yes

**Review by mark-antony1 — APPROVED** [review]
Looks great!

**Pull request merged**